namespaces/api-namespace.md

```diff
@@ -125,6 +140,15 @@ Given an API description with following layout:
 }
 ```
 
+## String Pair (object)
```

Comment:
A key in header can be used multiple times, so we need to be careful in deciding if we want to use this.